front/src/common-app/components/define-trend.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { cx } from 'emotion';
+import * as classes from './table-player.styles';
+import Typography from '@material-ui/core/Typography';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import { PlayerVotingStatus } from 'core';
+
+interface Props {
+  playersCollection: PlayerVotingStatus[];
+  headingLevel?: React.ElementType;
+}
+
+export const TablePlayerTrendComponent: React.FC<Props> = (props: Props) => {
+  const { playersCollection, headingLevel } = props;
+
+  const getTrend = (): string => {
+    const result = playersCollection.map((elem: PlayerVotingStatus) =>
+      playersCollection.filter(
+        (player: PlayerVotingStatus) => player.vote === elem.vote
+      ).length
+    );
+
+    const foundIndex = result.indexOf(Math.max(...result));
+
+    return playersCollection[foundIndex].vote;
```

Comment:
Some feedback, this is business code, let's extract it to the define-trend.business.ts

Instead of using map and filter, why not using redux and do this in one go

If you use reduce in the proper way you can create an struct like. {xs: 3, s: 4}